init-tools.cmd

```diff
@@ -77,6 +77,9 @@ if %UPDATE_CLI_ERRORLEVEL% GTR 1 (
   exit /b %UPDATE_CLI_ERRORLEVEL%
 )
 
+echo Copying supplemental overrides from Tools-Override.
+copy %~dp0Tools-Override\* %~dp0Tools
```

Comment:
I like the idea of using something like a Tools-Override directory to isolate everything we plan to move to buildtools.